android/clientlib-ktx/build.gradle

```diff
@@ -29,11 +29,16 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+
+    buildFeatures {
+        buildConfig = false
```

Comment:
Can you detail why this was needed?

Reply:
It was trying to generate BuildConfig, which was colliding with the core library one.